cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/SmartWorkflow.java

```diff
@@ -114,6 +112,8 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import static io.cdap.cdap.features.Feature.PIPELINE_COMPOSITE_TRIGGERS;
```

Comment:
style: for readability, we don't do static imports.

Import just the Feature class and refer to it as Feature.PIPELINE_COMPOSITE_TRIGGERS

Reply:
updated